packages/ui/src/Header.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Button } from './Button';
+import './header.css';
+
+interface HeaderProps {
+  user?: {};
+  onLogin: () => void;
+  onLogout: () => void;
+  onCreateAccount: () => void;
+}
```

Comment:
What component is this even? :/

Reply:
This component was the navBar component.